### PR TITLE
doc: Update LICENSE

### PR DESCRIPTION
**Pull request by Pjiesco**

### CODE REVIEW

**Comment by TinaTiel** [comment]
We could add the rest of the contributors here as well -- for me that'd be TinaTiel, with contact@tinatiel.com as the email.
@ChristopheCVB can we just use your contact email in your profile?

Finally, we should also add ourselves to the gradle build file as well.

**Comment by Pjiesco** [comment]
@TinaTiel yes, you are free to push your name to the branch.
I wanted to wait with the gradle build once we know what Maven Central repo and package name we are gonna use

File: build.gradle [hunk 1]
```diff
@@ -155,12 +156,12 @@ uploadArchives {
...
-                        name 'Lars Bärtschi'
-                        email 'lars@twasi.net'
+                        name 'Christophe Carvalho Vilas-Boas'
+                        email 'christophe.carvalhovilasboas@gmail.com'
                     }
                     developer {
-                        name 'Merlin Westphal'
-                        email 'merlin@twasi.net'
+                        name 'TinaTiel'
+                        email 'contact@tinatiel.com'
```
Comment (ChristopheCVB) [inline build.gradle:164]: @Pjiesco Maybe you can add yourself in here too

**Review by ChristopheCVB — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (Pjiesco) [reply build.gradle:164]: Yup will do

**Review by ChristopheCVB — APPROVED** [review]

**Pull request merged**